docassemble/AssemblyLine/al_document.py

```diff
@@ -989,7 +1027,7 @@ def send_button_html(self, key:str='final', show_editable_checkbox:bool = True)-
   <div class="al_email_container">
     <span class="al_email_address '''+name+''' form-group row da-field-container da-field-container-datatype-email">
       <label for="'''+al_email_input_id+'''" class="al_doc_email col-form-label da-form-label datext-right">Email</label>
-      <input value="''' + (user_info().email if user_logged_in() else '') + '''" alt="Input box" class="form-control" type="email" size="35" name="'''+al_email_input_id+'''" id="'''+al_email_input_id+'''">
+      <input value="''' + (user_info().email if user_logged_in() else '') + '" alt="Input box" class="form-control" type="email" size="35" name="'+al_email_input_id+'" id="'+al_email_input_id+'''">
```

Comment:
If we're going to adjust how these strings are put together, is this an appropriate place to starting using the `f"..."` format?

Reply:
Yes, I do think so. It would make it a lot easier to read this line. I think it can be its own PR though.